assets/src/stories-editor/components/block-navigation/edit.css

```diff
@@ -229,18 +229,27 @@
 	display: none;
 }
 
-.block-editor-block-navigation__list .block-editor-block-navigation__item .components-drop-zone.is-close-to-bottom {
+.block-editor-block-navigation__list li .block-editor-block-navigation__item .components-drop-zone {
 	background: none;
-	border-bottom: 3px solid #0071a1;
 }
 
-.block-editor-block-navigation__list li .block-editor-block-navigation__item .components-drop-zone.is-close-to-top {
-	background: none;
-	border: none;
+.block-editor-block-navigation__list li .block-navigation__placeholder {
+	display: none;
 }
 
-.block-editor-block-navigation__list li:first-child .block-editor-block-navigation__item .components-drop-zone.is-close-to-top {
-	background: none;
-	border-top: 3px solid #0071a1;
-	border-bottom: none;
+/* When dragging near the top of a drop zone, display the placeholder so there is a space between this and the button above. */
+.block-editor-block-navigation__list li .block-editor-block-navigation__item .components-drop-zone.is-close-to-top + .block-navigation__placeholder {
```

Comment:
It'd be nice if the  `div.block-navigation__placeholder` wasn't necessary. Maybe there's a way to avoid using it.

Its purpose is to add space between the buttons when dragging.

But when I tried to remove this style rule for the placeholder, and replace it with something like:

```css
.block-editor-block-navigation__list li .block-editor-block-navigation__item .components-drop-zone.is-close-to-top {
	position: static;
	height: 36px;
}
```

...there was a 'toggling' when dragging over the bottom button:

![last-drop](https://user-images.githubusercontent.com/4063887/62181435-e8eddb00-b318-11e9-8f6a-5da4d4dbd51c.gif)

Reply:
Probably the solution with the extra div is OK. If I'm not mistaken then this flickering might come from dragleave firing when moving over containers, however, not sure if this can be handled in a better way. You could look into `onDragLeave` event just in case or search about `dragleave` causing flickering, the placeholder might be a better option though :)

(Not 100% sure that the issue comes from that.)

Reply:
Good idea to look at the `onDragLeave` possibly causing the 'toggling.' I'll look at that.

Reply:
Good idea to check out the `dragleave` event.

I'm not sure if that's responsible for the 'flickering' above.

As a crude example, disabling it via:

```
document.querySelector( '#amp-story-block-navigation > div > div > ul > li:nth-last-of-type(1) > div > div.components-drop-zone' )
        .addEventListener(
                'dragleave', 
                function( event ) { event.preventDefault();
        }
);
```

...didn't seem to fix this. Though maybe disabling it isn't what you had in mind.

I think it might be caused by my [CSS snippet above](https://github.com/ampproject/amp-wp/pull/2920#discussion_r309025541) not being very good 😄 Though there might not be a way around it.

Reply:
Thanks for looking into this, it works well as it is, too.